### PR TITLE
Template Updates: Entertainment

### LINKED ISSUE #6972: Template Updates: Entertainment
## Description 

**Bolded* changes from April 6, 2021: https://docs.google.com/spreadsheets/d/1MFtHj7Xwdol9jSk80PXrxWNz_0h1tSfyEe3K7Bn2yZs/edit#gid=0

| Page | Animations | Cosmetics |
| --- | --- | --- |
| 1 |  Headline text changed to fly in. Remove animation on image. **Music Note to fade in (3000ms)**  | rename music shape? (needs clarification)  |
| 2 | headline and subheading text changed to fly in; remove animation on image.   | N/A |
| 3 |  **Everything should Fly in** | Remove one of the "pink" words in background, change position.  |
| 4 |  spotlight text updated to fly in; remove image animation. **Headline to fly in**  | N/A  |
| 5 | stars and ratings updated to fly in; remove image animation. **Text and Button to fade in (3000ms)**  | N/A |
| 6 | remove image animation; **Text to fly in**  | N/A |
| 7 | remove image animation ; **Text to fly in** | N/A  |
| 8 | **Headline to fly in; Images Fade in 3000ms; Text fly in**  | N/A |
| 9 |  Headline and subheading updated to fly in; remove image animation; **Grammys to Fade in (3000ms)** | N/A |

See Figma for specifications: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7216%3A549493 

If design requests have updated please make note in a comment here so that testing is aware of updated specs. 

## Acceptance 
- Template is updated 
- Relevant stickers are created 
- No more custom animations

### PR DESCRIPTION
**Pull request by embarks**
## Context
Updates the Entertainment template according to the table in the [ticket](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/6972)

## Summary
- Updates the animations on the elements so they use the editor animations
- Creates new stickers for various elements (rating stars and the music note)

## Relevant Technical Choices
NA

## To-do

- [x] Design review

## User-facing changes

https://user-images.githubusercontent.com/41136059/114254019-ea7efb80-9961-11eb-8f18-51a85c4a53e6.mp4

Screen Shot 2021-04-09 at 6 28 52 PM

## Testing Instructions
View the Entertainment template and compare the changes to [the ticket](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/6972)

### QA

- [x] This is a non-user-facing change and requires no QA

### UAT

- [x] UAT should use the same steps as above.

### Does this PR have a security-related impact?
NA

### Does this PR change what data or activity we track or use?
NA

### Does this PR have a legal-related impact?
NA

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6972

### CODE REVIEW

File: assets/src/edit-story/stickers/entertainment5Star.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+function Entertainment5Star({ style }) {
+  return (
+    <svg
+      style={style}
+      width="17"
+      height="15"
```
Comment (maxyinger) [inline assets/src/edit-story/stickers/entertainment5Star.js]: rm height and width here

File: assets/src/edit-story/stickers/entertainment1Music.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+function Entertainment1Music({ style }) {
+  return (
+    <svg
+      style={style}
+      width="160"
+      height="156"
```
Comment (maxyinger) [inline assets/src/edit-story/stickers/entertainment1Music.js]: rm height and width here

**Review by maxyinger — APPROVED** [review]
Looks great! Very smart move with the layers to get a reveal like effect 🎉 

I think just removing the height and width attributes from the stickers and should be good. Although... even with the height and width attributes on there stickers seem to be resizing fine. thought I was running into issues last I tried this, but maybe we don't need to remove them after all 🤷

**Comment by BrittanyIRL** [comment]
@googlebot I consent.

**Review by BrittanyIRL — APPROVED** [review]

**Comment by ivayloklissarov** [comment]
Small fixes:

General
- Add or change to fade in 2000ms animation for all images on each page

Page 1
- Change entering position to right for text in white
- Change entering position to left for text in grey
Screen Shot 2021-04-15 at 9 20 30 AM

Page 2
- Remove animation of "artists" text
Screen Shot 2021-04-15 at 9 25 42 AM

Page 3
- Remove animation for the commas
- Remove animation for the "artist" text
- Change entering position to right for big text "pink" in the background
Screen Shot 2021-04-15 at 9 23 23 AM

Page 4
- Change entering position to left for "spotlight" text in white
- Remove animation of "spotlight" text in grey
Screen Shot 2021-04-15 at 9 26 57 AM

Page 6
- Remove animation for text on top
Screen Shot 2021-04-15 at 9 28 36 AM

Page 7
- Remove animation of title
Screen Shot 2021-04-15 at 9 34 03 AM

Page 8 
- Remove animation of title
Screen Shot 2021-04-15 at 9 39 32 AM

**Comment by BrittanyIRL** [comment]
> Small fixes:
> 
> General
> 
> * Add or change to fade in 2000ms animation for all images on each page
> 
> Page 1
> 
> * Change entering position to right for text in white
> * Change entering position to left for text in grey

entertainment page 1

> Page 2
> 
> * Remove animation of "artists" text

entertainment page 2

> Page 3
> 
> * Remove animation for the commas
> * Remove animation for the "artist" text
> * Change entering position to right for big text "pink" in the background

entertainment page 3

> Page 4
> 
> * Change entering position to left for "spotlight" text in white
> * Remove animation of "spotlight" text in grey

entertainment page 4


> Page 6
> 
> * Remove animation for text on top

entertainment page 6

> Page 7
> 
> * Remove animation of title

entertainment page 7

> Page 8
> 
> * Remove animation of title

entertainment page 8

**Pull request merged**